pkg/services/notifications/webhook.go

```diff
@@ -32,21 +29,6 @@ type WebhookClient interface {
 	Do(req *http.Request) (*http.Response, error)
 }
 
-var netTransport = &http.Transport{
-	TLSClientConfig: &tls.Config{
-		Renegotiation: tls.RenegotiateFreelyAsClient,
-	},
-	Proxy: http.ProxyFromEnvironment,
-	Dial: (&net.Dialer{
-		Timeout: 30 * time.Second,
-	}).Dial,
-	TLSHandshakeTimeout: 5 * time.Second,
-}
-var netClient WebhookClient = &http.Client{
-	Timeout:   time.Second * 30,
```

Comment:
Moved as is to pkg/services/notifications/notifications.go, with the exception of this line which I used the setting read from the configuration file.